Meetups/25/2025-03-14-minutes.md

```diff
@@ -0,0 +1,181 @@
+# Meeting Minutes
+
+:date: **Date:** 14 March 2025
+
+### :bust_in_silhouette: Participants
+
+<!-- This list will copied over from the meeting tool -->
+- Ege Korkan
+- Cristiano Aguzzi
+- Robert Winkler
+- CodingNinja
+- Samuele Marro
+- DB
+- daniel o
+- Rem Collier
+- Jochen Hiller
+- Katharine Beaumont
+- Denis
+- Andrei Olaru
+- Thomas Jaeckle
+- Christian Glomb
+- Andrei Ciortea
+- Jonathan Rau
+- Vignesh V
+- Maud Stiernet
+- Kunihiko Toumura
+- Roman Binkert
+- Henning Sudbrock
+- Joshua Cornejo
+- Anastasiya Danilenka
+- L. Gustavo Nardin
+- Andrea Giulianelli
+- Tomoaki Mizushima
+- Jasbir Singh
+- Kai Kreuzer
+- Ben Francis
+- Nikita
+- Jean-Paul Calbimonte
+- A43581519
+- Sam
+- Stephen Cranefield
+- Tatiana
+- Daniel Peintner
+- Alexandru Sorici
+- Terry Payne
+- Julian Padget
+- Stefan Bußweiler
+- Raul
+- Patrick
+- Lorenzo Gigli
+- Gianluca Aguzzi
+- Alessandro Giugno
+
+:writing_hand: **Scribe:** Cris
+
+----
+
+### Introduction
+
+- Ege: welcome everyone! This is a joint call between WoT, WebAgents and Web Thing Protocol CG
+- Ege: Working at siemens and co-lead of CG group
+- Cris: Work in standards and in a company called Archeion.
+- Andrei: Web agents CG
+- Rem: Professor working with multi agents system, involved in hypermedia multi agents systems
+- Ben: Founder of Krellian, Web Thing Protocol CG would ask anyone who is interested in the development of the protocol to join the group.
+
+### News
+
+Ege:two weeks from now we will host a new meetup about HiveOT
+
+Ege: More are planned
+
+Ege: if you anything to present let us know!
+
+Andrei: Quick news from us, we are organizing a workshop. If you have research or innovation projects please reach out.
+
+Ben: We have started writing the specification of Web Thing Protocol and there is a strawman proposal (watch out is getting out of date).
+
+Ege: Joining the groups is free, you can join using the button.
+
+### Meetup Presentation
+
+Ege: Today we have Robert Winkler one of the Driver of LMOS project. He is focused on internet of Things and conversational AI. Solution architect at day and a gamer at night.
+
+Robert: We will talk about LMOS at Eclipse but also how we used WoT in that system
+
+Robert: At DT we are bring GEnAi to solve sales and services. We are going soon to integrate voice in the system. That solution is based on a LLM. You can augment LLMs with tools, usually functions to performing actions in the digital/physical world. The agent can also access short and long term memory.
+
+Robert: the focus of today is more about the process we went trough when building such system and where WoT played a crucial role.
+
+Robert: We split the problem in different agents to improve performance (one agent was performing bad).
+
+Robert: Now we don't allow communication between agents, main challenge here is to avoid loops. At DT we are switching from API to agents. Soon we'll have an internet of agents not API calls.
```

Comment:
```suggestion
Robert: We split the problem over different agents to improve performance (one agent was performing poorly).

Robert: Now we don't allow communication between agents; main challenge is to avoid loops. At DT, we are switching from API to agents. Soon we'll have an internet of agents, not API calls.
```